routes/user.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+const passport = require('../config/ppConfig')
+const db = require("../models")
+const methodOverride = require('method-override');
+
+router.use(methodOverride('_method'));
+//Get "/"
+router.get()
+//Post "/"
```

Comment:
should it be POST for `/meeting` or `/meeting/new` for a creating a new meeting?